### PR TITLE
Adding a new QS for Marketing Data Foundation Starter V2 Solution

### PR DESCRIPTION
**Pull request by sfc-gh-mfigueroamontero**
This new QS will provide you with the steps required to deploy the Marketing Data Foundation Starter V2
This sample is part of the [Marketing Data Foundation Solution](https://github.com/Snowflake-Labs/sfguide-marketing-data-foundation-starter)

### CODE REVIEW

File: site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Marketing, Data Engineering, Native Application
+
+# Marketing Data Foundation For Customer 360 Starter Guide
+<!-- ------------------------ -->
+## Introduction 
+
+Duration: 15
+
+Customers looking to use Snowflake for marketing use cases initially face a significant challenge: it is difficult to import all relevant marketing data into Snowflake and structure it in a unified format that downstream applications can easily utilize to power data-driven marketing.
```
Comment (sfc-gh-jacunasolano) [inline site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md]:
```suggestion
Customers looking to use Snowflake for marketing use cases initially face a significant challenge, it is difficult to import all relevant marketing data into Snowflake and structure it in a unified format that downstream applications can easily utilize to power data-driven marketing.
```

File: site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+This starter solution tackles this challenge of creating a marketing data foundation for customer 360.
+This is part of the overall [Marketing Data Foundation Starter]() solution.
+
+### Problem Space
+
+The Customer 360 challenge revolves around creating a unified, comprehensive view of a person, whether a customer or prospect. This endeavor involves aggregating and synthesizing data from a multitude of touchpoints and interactions customers have with a brand, including online behaviors, transaction histories, and customer service experiences. The main challenge is data siloing and integration, as information often resides in disparate systems, making it difficult to achieve a singular, holistic view of the customer.
```
Comment (sfc-gh-jacunasolano) [inline site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md]:
```suggestion
The Customer 360 challenge revolves around creating a unified and comprehensive view of a person, whether it's a customer or prospect. This endeavor involves aggregating and synthesizing data from a multitude of touchpoints and interactions that customers have with a brand, including: online behaviors, transaction histories, and customer service experiences. The main challenge is data siloing and integration, as information often resides in disparate systems, making it difficult to achieve a singular and holistic view of the customer.
```

File: site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+
+More specifically, Customer 360 includes the following data types:
```
Comment (sfc-gh-jacunasolano) [inline site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md]:
```suggestion
More precisely, Customer 360 includes the following data types:
```

File: site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+
+* <b>Demographic Data:</b> Information such as age, gender, income, education, and marital status that helps in segmenting the customer base.<br>
+* <b>Transactional Data:</b> Records of customer purchases, returns, and exchanges, including details like transaction amounts, product categories, and purchase history.<br>
```
Comment (sfc-gh-jacunasolano) [inline site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md]:
```suggestion
* <b>Transactional Data:</b> Records of customer purchases, returns, and exchanges; including details like transaction amounts, product categories, and purchase history.<br>
```

File: site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+* <b>Behavioral Data:</b> Data on customer interactions with the brand, including website visits, app usage, click-through rates, and engagement with marketing and loyalty campaigns.<br>
```
Comment (sfc-gh-jacunasolano) [inline site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md]:
```suggestion
* <b>Behavioral Data:</b> Data on customer interactions with the brand, including: website visits, app usage, click-through rates, and engagement with marketing and loyalty campaigns.<br>
```

File: site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+* <b>Engagement Data:</b> Details of how customers engage with the brand through various channels, such as email open rates, social media interactions, and customer service inquiries.<br>
```
Comment (sfc-gh-jacunasolano) [inline site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md]:
```suggestion
* <b>Engagement Data:</b> Details of how customers engage with the brand through various channels, such as: email open rates, social media interactions, and customer service inquiries.<br>
```

File: site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md [hunk 7]
```diff
@@ -0,0 +1,211 @@
...
+* <b>Psychographic Data:</b> Insights into the customer's lifestyle, interests, attitudes, and opinions, often gathered through surveys, feedback forms, and social media analysis.<br>
+* <b>Customer Feedback:</b> Direct feedback from customers, including Net Promoter Score (NPS) surveys, customer satisfaction surveys, reviews, and complaints.
```
Comment (sfc-gh-jacunasolano) [inline site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md]:
```suggestion
* <b>Customer Feedback:</b> Direct feedback from customers, including: Net Promoter Score (NPS) surveys, customer satisfaction surveys, reviews, and complaints.
```

File: site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md [hunk 8]
```diff
@@ -0,0 +1,211 @@
...
+* <b>Customer Service Platforms:</b> Data from customer service interactions, including support tickets, chat transcripts, and call logs, provide insights into customer issues, preferences, and feedback. (E.g., Zendesk, Salesforce Service Cloud, Freshdesk)<br>
+* <b>Survey Tools and Feedback Platforms:</b> These sources capture direct customer feedback, opinions, and satisfaction levels, which are crucial for understanding customer sentiment. (E.g., SurveyMonkey, Qualtrics, Google Forms)<br>
+* <b>Point of Sale (POS) Systems:</b> For brick-and-mortar operations, POS systems capture in-store transactions, returns, and customer interactions. (E.g., Square, Clover, Lightspeed)<br>
+
+Data privacy and security are paramount concerns in building a Customer 360 view. With increasing scrutiny on how companies collect, store, and use customer data, navigating privacy regulations such as GDPR and CCPA becomes a complex but critical task. This involves ensuring all customer data is handled in a compliant manner, balancing the need for insightful analytics with the imperative of protecting customer privacy.
```
Comment (sfc-gh-jacunasolano) [inline site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md]:
```suggestion
Data privacy and security are paramount concerns in building a Customer 360 view. With increasing scrutiny on how companies collect, store, and use customer data; navigating privacy regulations such as GDPR and CCPA becomes a complex but critical task. This involves ensuring all customer data is handled in a compliant manner, balancing the need for insightful analytics with the imperative of protecting customer privacy.
```

File: site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md [hunk 9]
```diff
@@ -0,0 +1,211 @@
...
+
+### Solution Space
+
+#### Context
+
+As described in the diagram below, the two Data Foundation use cases in this starter lay the groundwork to support the two Marketing Execution use cases: Planning & Activation, and Measurement.
```
Comment (sfc-gh-jacunasolano) [inline site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md]:
```suggestion
As described in the diagram below, the two Data Foundation use cases in this starter lay the foundation to support the two Marketing Execution use cases: Planning & Activation, and Measurement.
```

File: site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md [hunk 10]
```diff
@@ -0,0 +1,211 @@
...
+```console
+snow app run
+```
+<img src="assets/Appcreation.png" alt="conn-info" height="600" width="800"/>
+
+
+## Conclusion
+Duration: 1
+
+Congratulations! You have successfully learnt how to easily build an end to end Native Application and loading sample data.
```
Comment (sfc-gh-jacunasolano) [inline site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md]:
```suggestion
Congratulations! You have successfully learn how to easily build an end to end Native Application and load some sample data. 
```

File: site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md [hunk 11]
```diff
@@ -0,0 +1,211 @@
...
+![](assets/ApplicationListed.png)
+
+If you open the app and navigate to the the Customer 360 starter tab, you will see some infographics describing the sample data we just uploaded.
+![](assets/AppScreenshot.png)
+
+### What you learned
+
+* How to host build a basic native app for Customer 360 Marketing data.
+* Generate a set of views that aggregate data from Marketing providers like Google and Salesforce.
+* Generate quick summaries and Charts using that data to create an integrated dashboard using streamlit.
```
Comment (sfc-gh-jacunasolano) [inline site/sfguides/src/marketing_data_foundation_customer_360/marketing_data_foundation_customer_360.md]:
```suggestion
* How to host and build a basic native app for Customer 360 Marketing data.
* Generate a set of views that aggregate data from Marketing providers like Google and Salesforce.
* Generate quick summaries and charts using that data to create an integrated dashboard using Streamlit.
```

**Comment by sfc-gh-mfigueroamontero** [comment]
image

**Pull request merged**